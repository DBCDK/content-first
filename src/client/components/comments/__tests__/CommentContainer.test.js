import React from 'react';
import {CommentContainer} from '../Comment.container';
import renderer from 'react-test-renderer';
import {mount} from 'enzyme';

jest.mock('react-textarea-autosize', () => 'textarea');
jest.mock('../CommentWrapper.component.js', () => 'comment');

function generateComments(count) {
  const comments = [];
  for (let i = 0; i < count; i++) {
    comments.push({
      comment: `comment${i}`,
      _id: `${i}`,
      user: {
        name: 'Benny Cosmos'
      }
    });
  }
  return comments;
}

describe('CommentContainer', () => {
  test('Component fetches comments on mount', () => {
    const fetch = jest.fn();
    const tree = renderer
      .create(
        <CommentContainer
          id="some_id"
          fetchComments={fetch}
          comments={[]}
          user={{image: 'image_id'}}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
    expect(fetch.mock.calls[0][0]).toEqual('some_id');
  });
  test('Component renders a list of comments', () => {
    const tree = renderer
      .create(
        <CommentContainer
          id="some_id"
          fetchComments={jest.fn()}
          comments={generateComments(2)}
          user={{image: 'image_id'}}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('Component renders error', () => {
    const tree = mount(
      <CommentContainer
        id="some_id"
        fetchComments={jest.fn()}
        error={{
          comment: 'comment3',
          error: 'some error occures'
        }}
        comments={generateComments(2)}
        user={{image: 'image_id'}}
      />
    );
    tree.setState({newCommentValue: 'comment3'});
    expect(tree).toMatchSnapshot();
  });
  test('Component renders saving overlay', () => {
    const tree = mount(
      <CommentContainer
        id="some_id"
        fetchComments={jest.fn()}
        comments={[
          ...generateComments(2),
          {
            comment: 'comment new',
            saving: true,
            _id: 'new_comment',
            user: {name: 'Benny Cosmos'}
          }
        ]}
        user={{image: 'image_id'}}
      />
    );
    expect(tree).toMatchSnapshot();
  });
  test('Component renders all comments', () => {
    const tree = mount(
      <CommentContainer
        id="some_id"
        fetchComments={jest.fn()}
        comments={generateComments(5)}
        user={{image: 'image_id'}}
      />
    );
    tree.setState({showAll: true});
    expect(tree).toMatchSnapshot();
  });
  test('Component renders comment with links and emojis', () => {
    const comment = {
      comment: 'This is a comment with ❤️ and links.com \n and line breaks',
      _id: 'comment',
      user: {name: 'Benny Cosmos'}
    };
    const tree = renderer.create(
      <CommentContainer
        id="some_id"
        fetchComments={jest.fn()}
        comments={[comment]}
        user={{image: 'image_id'}}
      />
    );
    expect(tree).toMatchSnapshot();
  });
  test('Component Submits a new comment', () => {
    const addComment = jest.fn();
    const tree = mount(
      <CommentContainer
        id="some_id"
        fetchComments={jest.fn()}
        addComment={addComment}
        comments={[]}
        user={{image: 'image_id', openplatformId: 'owner_id'}}
      />
    );
    tree.setState({newCommentValue: 'a new comment'});
    tree
      .find('#comment-submit')
      .hostNodes()
      .simulate('click');

    expect(addComment.mock.calls[0]).toEqual([
      {id: 'some_id', comment: 'a new comment', owner: 'owner_id'}
    ]);
  });
  test('Toggle all comments', () => {
    const tree = mount(
      <CommentContainer
        id="some_id"
        fetchComments={jest.fn()}
        comments={generateComments(5)}
        user={{image: 'image_id'}}
      />
    );
    const toggleButton = tree.find('#comment-toggle');
    expect(tree).toMatchSnapshot();
    expect(toggleButton.text()).toContain('Vis alle kommentarer');
    toggleButton.simulate('click');
    expect(toggleButton.text()).toContain('Vis færre kommentarer');
    expect(tree).toMatchSnapshot();
  });
});
