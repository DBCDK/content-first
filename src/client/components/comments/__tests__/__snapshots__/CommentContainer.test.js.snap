// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentContainer Component fetches comments on mount 1`] = `
<div
  className="comments"
/>
`;

exports[`CommentContainer Component renders a list of comments 1`] = `
<div
  className="comments"
>
  <div
    className="mb3"
  >
    <div
      style={
        Object {
          "height": "0px",
          "overflow": "hidden",
          "transition": "height 500ms",
        }
      }
    >
      <div>
        <comment
          comment={
            Object {
              "_id": "1",
              "comment": "comment1",
              "user": Object {
                "name": "Benny Cosmos",
              },
            }
          }
          onChange={[Function]}
        />
      </div>
    </div>
    <button
      className="btn btn-link mt1 mb1 link-subtle"
      id="comment-toggle"
      onClick={[Function]}
      style={
        Object {
          "marginLeft": 55,
          "paddingLeft": 0,
          "position": "relative",
        }
      }
    >
      <span
        className={undefined}
        style={
          Object {
            "fontFamily": "var(--secondary-font)",
            "fontSize": 10,
            "position": "relative",
          }
        }
      >
        <span
          className="glyphicon glyphicon-comment"
          style={
            Object {
              "fontSize": 24,
              "verticalAlign": "middle",
            }
          }
        />
        <span
          className="number"
          style={
            Object {
              "color": "white",
              "fontSize": 11,
              "fontWeight": "normal",
              "left": 0,
              "position": "absolute",
              "textAlign": "center",
              "top": "50%",
              "transform": "translateY(-50%)",
              "width": "100%",
            }
          }
        >
          2
        </span>
      </span>
      <span
        className="ml1"
      >
        Vis alle kommentarer
      </span>
    </button>
  </div>
</div>
`;

exports[`CommentContainer Component renders all comments 1`] = `
<CommentContainer
  comments={
    Array [
      Object {
        "_id": "0",
        "comment": "comment0",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "1",
        "comment": "comment1",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "2",
        "comment": "comment2",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "3",
        "comment": "comment3",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "4",
        "comment": "comment4",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
    ]
  }
  fetchComments={[Function]}
  id="some_id"
  user={
    Object {
      "image": "image_id",
    }
  }
>
  <div
    className="comments"
  >
    <div
      className="mb3"
    >
      <CommentList
        comments={
          Array [
            Object {
              "_id": "0",
              "comment": "comment0",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "1",
              "comment": "comment1",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "2",
              "comment": "comment2",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "3",
              "comment": "comment3",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "4",
              "comment": "comment4",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
          ]
        }
        showCount={5}
        user={
          Object {
            "image": "image_id",
          }
        }
      >
        <div
          style={
            Object {
              "height": "0px",
              "overflow": "hidden",
              "transition": "height 500ms",
            }
          }
        >
          <div>
            <comment
              comment={
                Object {
                  "_id": "0",
                  "comment": "comment0",
                  "user": Object {
                    "name": "Benny Cosmos",
                  },
                }
              }
              key="0"
              onChange={[Function]}
            />
            <comment
              comment={
                Object {
                  "_id": "1",
                  "comment": "comment1",
                  "user": Object {
                    "name": "Benny Cosmos",
                  },
                }
              }
              key="1"
              onChange={[Function]}
            />
            <comment
              comment={
                Object {
                  "_id": "2",
                  "comment": "comment2",
                  "user": Object {
                    "name": "Benny Cosmos",
                  },
                }
              }
              key="2"
              onChange={[Function]}
            />
            <comment
              comment={
                Object {
                  "_id": "3",
                  "comment": "comment3",
                  "user": Object {
                    "name": "Benny Cosmos",
                  },
                }
              }
              key="3"
              onChange={[Function]}
            />
            <comment
              comment={
                Object {
                  "_id": "4",
                  "comment": "comment4",
                  "user": Object {
                    "name": "Benny Cosmos",
                  },
                }
              }
              key="4"
              onChange={[Function]}
            />
          </div>
        </div>
      </CommentList>
      <button
        className="btn btn-link mt1 mb1 link-subtle"
        id="comment-toggle"
        onClick={[Function]}
        style={
          Object {
            "marginLeft": 55,
            "paddingLeft": 0,
            "position": "relative",
          }
        }
      >
        <Component
          value={5}
        >
          <span
            style={
              Object {
                "fontFamily": "var(--secondary-font)",
                "fontSize": 10,
                "position": "relative",
              }
            }
          >
            <span
              className="glyphicon glyphicon-comment"
              style={
                Object {
                  "fontSize": 24,
                  "verticalAlign": "middle",
                }
              }
            />
            <span
              className="number"
              style={
                Object {
                  "color": "white",
                  "fontSize": 11,
                  "fontWeight": "normal",
                  "left": 0,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": "50%",
                  "transform": "translateY(-50%)",
                  "width": "100%",
                }
              }
            >
              5
            </span>
          </span>
        </Component>
        <span
          className="ml1"
        >
          Vis færre kommentarer
        </span>
      </button>
    </div>
    <CommentInput
      error={null}
      onCancel={[Function]}
      onChange={[Function]}
      onSubmit={[Function]}
      user={
        Object {
          "image": "image_id",
        }
      }
      value=""
    />
  </div>
</CommentContainer>
`;

exports[`CommentContainer Component renders comment with links and emojis 1`] = `
<div
  className="comments"
>
  <div
    className="mb3"
  >
    <div
      style={
        Object {
          "height": "0px",
          "overflow": "hidden",
          "transition": "height 500ms",
        }
      }
    >
      <div>
        <comment
          comment={
            Object {
              "_id": "comment",
              "comment": "This is a comment with ❤️ and links.com 
             and line breaks",
              "user": Object {
                "name": "Benny Cosmos",
              },
            }
          }
          onChange={[Function]}
        />
      </div>
    </div>
    
  </div>
</div>
`;

exports[`CommentContainer Component renders error 1`] = `
<CommentContainer
  comments={
    Array [
      Object {
        "_id": "0",
        "comment": "comment0",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "1",
        "comment": "comment1",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
    ]
  }
  error={
    Object {
      "comment": "comment3",
      "error": "some error occures",
    }
  }
  fetchComments={[Function]}
  id="some_id"
  user={
    Object {
      "image": "image_id",
    }
  }
>
  <div
    className="comments"
  >
    <div
      className="mb3"
    >
      <CommentList
        comments={
          Array [
            Object {
              "_id": "0",
              "comment": "comment0",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "1",
              "comment": "comment1",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
          ]
        }
        showCount={1}
        user={
          Object {
            "image": "image_id",
          }
        }
      >
        <div
          style={
            Object {
              "height": "0px",
              "overflow": "hidden",
              "transition": "height 500ms",
            }
          }
        >
          <div>
            <comment
              comment={
                Object {
                  "_id": "1",
                  "comment": "comment1",
                  "user": Object {
                    "name": "Benny Cosmos",
                  },
                }
              }
              key="1"
              onChange={[Function]}
            />
          </div>
        </div>
      </CommentList>
      <button
        className="btn btn-link mt1 mb1 link-subtle"
        id="comment-toggle"
        onClick={[Function]}
        style={
          Object {
            "marginLeft": 55,
            "paddingLeft": 0,
            "position": "relative",
          }
        }
      >
        <Component
          value={2}
        >
          <span
            style={
              Object {
                "fontFamily": "var(--secondary-font)",
                "fontSize": 10,
                "position": "relative",
              }
            }
          >
            <span
              className="glyphicon glyphicon-comment"
              style={
                Object {
                  "fontSize": 24,
                  "verticalAlign": "middle",
                }
              }
            />
            <span
              className="number"
              style={
                Object {
                  "color": "white",
                  "fontSize": 11,
                  "fontWeight": "normal",
                  "left": 0,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": "50%",
                  "transform": "translateY(-50%)",
                  "width": "100%",
                }
              }
            >
              2
            </span>
          </span>
        </Component>
        <span
          className="ml1"
        >
          Vis alle kommentarer
        </span>
      </button>
    </div>
    <CommentInput
      error={
        Object {
          "comment": "comment3",
          "error": "some error occures",
        }
      }
      onCancel={[Function]}
      onChange={[Function]}
      onSubmit={[Function]}
      user={
        Object {
          "image": "image_id",
        }
      }
      value="comment3"
    />
  </div>
</CommentContainer>
`;

exports[`CommentContainer Component renders saving overlay 1`] = `
<CommentContainer
  comments={
    Array [
      Object {
        "_id": "0",
        "comment": "comment0",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "1",
        "comment": "comment1",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "new_comment",
        "comment": "comment new",
        "saving": true,
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
    ]
  }
  fetchComments={[Function]}
  id="some_id"
  user={
    Object {
      "image": "image_id",
    }
  }
>
  <div
    className="comments"
  >
    <div
      className="mb3"
    >
      <CommentList
        comments={
          Array [
            Object {
              "_id": "0",
              "comment": "comment0",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "1",
              "comment": "comment1",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "new_comment",
              "comment": "comment new",
              "saving": true,
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
          ]
        }
        showCount={1}
        user={
          Object {
            "image": "image_id",
          }
        }
      >
        <div
          style={
            Object {
              "height": "0px",
              "overflow": "hidden",
              "transition": "height 500ms",
            }
          }
        >
          <div>
            <comment
              comment={
                Object {
                  "_id": "new_comment",
                  "comment": "comment new",
                  "saving": true,
                  "user": Object {
                    "name": "Benny Cosmos",
                  },
                }
              }
              key="new_comment"
              onChange={[Function]}
            />
          </div>
        </div>
      </CommentList>
      <button
        className="btn btn-link mt1 mb1 link-subtle"
        id="comment-toggle"
        onClick={[Function]}
        style={
          Object {
            "marginLeft": 55,
            "paddingLeft": 0,
            "position": "relative",
          }
        }
      >
        <Component
          value={3}
        >
          <span
            style={
              Object {
                "fontFamily": "var(--secondary-font)",
                "fontSize": 10,
                "position": "relative",
              }
            }
          >
            <span
              className="glyphicon glyphicon-comment"
              style={
                Object {
                  "fontSize": 24,
                  "verticalAlign": "middle",
                }
              }
            />
            <span
              className="number"
              style={
                Object {
                  "color": "white",
                  "fontSize": 11,
                  "fontWeight": "normal",
                  "left": 0,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": "50%",
                  "transform": "translateY(-50%)",
                  "width": "100%",
                }
              }
            >
              3
            </span>
          </span>
        </Component>
        <span
          className="ml1"
        >
          Vis alle kommentarer
        </span>
      </button>
    </div>
    <CommentInput
      error={null}
      onCancel={[Function]}
      onChange={[Function]}
      onSubmit={[Function]}
      user={
        Object {
          "image": "image_id",
        }
      }
      value=""
    />
  </div>
</CommentContainer>
`;

exports[`CommentContainer Toggle all comments 1`] = `
<CommentContainer
  comments={
    Array [
      Object {
        "_id": "0",
        "comment": "comment0",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "1",
        "comment": "comment1",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "2",
        "comment": "comment2",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "3",
        "comment": "comment3",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "4",
        "comment": "comment4",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
    ]
  }
  fetchComments={[Function]}
  id="some_id"
  user={
    Object {
      "image": "image_id",
    }
  }
>
  <div
    className="comments"
  >
    <div
      className="mb3"
    >
      <CommentList
        comments={
          Array [
            Object {
              "_id": "0",
              "comment": "comment0",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "1",
              "comment": "comment1",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "2",
              "comment": "comment2",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "3",
              "comment": "comment3",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "4",
              "comment": "comment4",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
          ]
        }
        showCount={1}
        user={
          Object {
            "image": "image_id",
          }
        }
      >
        <div
          style={
            Object {
              "height": "0px",
              "overflow": "hidden",
              "transition": "height 500ms",
            }
          }
        >
          <div>
            <comment
              comment={
                Object {
                  "_id": "4",
                  "comment": "comment4",
                  "user": Object {
                    "name": "Benny Cosmos",
                  },
                }
              }
              key="4"
              onChange={[Function]}
            />
          </div>
        </div>
      </CommentList>
      <button
        className="btn btn-link mt1 mb1 link-subtle"
        id="comment-toggle"
        onClick={[Function]}
        style={
          Object {
            "marginLeft": 55,
            "paddingLeft": 0,
            "position": "relative",
          }
        }
      >
        <Component
          value={5}
        >
          <span
            style={
              Object {
                "fontFamily": "var(--secondary-font)",
                "fontSize": 10,
                "position": "relative",
              }
            }
          >
            <span
              className="glyphicon glyphicon-comment"
              style={
                Object {
                  "fontSize": 24,
                  "verticalAlign": "middle",
                }
              }
            />
            <span
              className="number"
              style={
                Object {
                  "color": "white",
                  "fontSize": 11,
                  "fontWeight": "normal",
                  "left": 0,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": "50%",
                  "transform": "translateY(-50%)",
                  "width": "100%",
                }
              }
            >
              5
            </span>
          </span>
        </Component>
        <span
          className="ml1"
        >
          Vis alle kommentarer
        </span>
      </button>
    </div>
    <CommentInput
      error={null}
      onCancel={[Function]}
      onChange={[Function]}
      onSubmit={[Function]}
      user={
        Object {
          "image": "image_id",
        }
      }
      value=""
    />
  </div>
</CommentContainer>
`;

exports[`CommentContainer Toggle all comments 2`] = `
<CommentContainer
  comments={
    Array [
      Object {
        "_id": "0",
        "comment": "comment0",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "1",
        "comment": "comment1",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "2",
        "comment": "comment2",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "3",
        "comment": "comment3",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
      Object {
        "_id": "4",
        "comment": "comment4",
        "user": Object {
          "name": "Benny Cosmos",
        },
      },
    ]
  }
  fetchComments={[Function]}
  id="some_id"
  user={
    Object {
      "image": "image_id",
    }
  }
>
  <div
    className="comments"
  >
    <div
      className="mb3"
    >
      <CommentList
        comments={
          Array [
            Object {
              "_id": "0",
              "comment": "comment0",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "1",
              "comment": "comment1",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "2",
              "comment": "comment2",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "3",
              "comment": "comment3",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
            Object {
              "_id": "4",
              "comment": "comment4",
              "user": Object {
                "name": "Benny Cosmos",
              },
            },
          ]
        }
        showCount={5}
        user={
          Object {
            "image": "image_id",
          }
        }
      >
        <div
          style={
            Object {
              "height": "0px",
              "overflow": "hidden",
              "transition": "height 500ms",
            }
          }
        >
          <div>
            <comment
              comment={
                Object {
                  "_id": "0",
                  "comment": "comment0",
                  "user": Object {
                    "name": "Benny Cosmos",
                  },
                }
              }
              key="0"
              onChange={[Function]}
            />
            <comment
              comment={
                Object {
                  "_id": "1",
                  "comment": "comment1",
                  "user": Object {
                    "name": "Benny Cosmos",
                  },
                }
              }
              key="1"
              onChange={[Function]}
            />
            <comment
              comment={
                Object {
                  "_id": "2",
                  "comment": "comment2",
                  "user": Object {
                    "name": "Benny Cosmos",
                  },
                }
              }
              key="2"
              onChange={[Function]}
            />
            <comment
              comment={
                Object {
                  "_id": "3",
                  "comment": "comment3",
                  "user": Object {
                    "name": "Benny Cosmos",
                  },
                }
              }
              key="3"
              onChange={[Function]}
            />
            <comment
              comment={
                Object {
                  "_id": "4",
                  "comment": "comment4",
                  "user": Object {
                    "name": "Benny Cosmos",
                  },
                }
              }
              key="4"
              onChange={[Function]}
            />
          </div>
        </div>
      </CommentList>
      <button
        className="btn btn-link mt1 mb1 link-subtle"
        id="comment-toggle"
        onClick={[Function]}
        style={
          Object {
            "marginLeft": 55,
            "paddingLeft": 0,
            "position": "relative",
          }
        }
      >
        <Component
          value={5}
        >
          <span
            style={
              Object {
                "fontFamily": "var(--secondary-font)",
                "fontSize": 10,
                "position": "relative",
              }
            }
          >
            <span
              className="glyphicon glyphicon-comment"
              style={
                Object {
                  "fontSize": 24,
                  "verticalAlign": "middle",
                }
              }
            />
            <span
              className="number"
              style={
                Object {
                  "color": "white",
                  "fontSize": 11,
                  "fontWeight": "normal",
                  "left": 0,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": "50%",
                  "transform": "translateY(-50%)",
                  "width": "100%",
                }
              }
            >
              5
            </span>
          </span>
        </Component>
        <span
          className="ml1"
        >
          Vis færre kommentarer
        </span>
      </button>
    </div>
    <CommentInput
      error={null}
      onCancel={[Function]}
      onChange={[Function]}
      onSubmit={[Function]}
      user={
        Object {
          "image": "image_id",
        }
      }
      value=""
    />
  </div>
</CommentContainer>
`;
